Assets/MixedRealityToolkit.Services/InputAnimation/IMixedRealityInputPlaybackService.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Plays back input animation via the input simulation system.
+    /// </summary>
+    public interface IMixedRealityInputPlaybackService : IMixedRealityInputDeviceManager
+    {
+        /// <summary>
+        /// The animation currently being played.
+        /// </summary>
+        InputAnimation Animation { get; set; }
+
+        /// <summary>
+        /// True if the animation is currently playing.
+        /// </summary>
+        bool IsPlaying { get; }
+
+        /// <summary>
+        /// The local time relative to the start of the animation.
```

Comment:
Time is always a tricky thing - it would be helpful to know what the units are - i.e. are these milliseconds since epoch? Milliseconds since player start?

Reply:
Improved comments.